.github/workflows/evm-version-compatibility-tests.yml

```diff
@@ -55,11 +54,75 @@ jobs:
             echo "dependency_changed=false" >> $GITHUB_OUTPUT
           fi
 
+  # Check if images used as test dependencies exist in the ECR 
+  check-ecr-images-exist:
+    if: needs.check-dependency-bump.outputs.dependency_changed == 'true' || github.event_name == 'workflow_dispatch'
+    needs: [check-dependency-bump]
+    environment: integration
+    permissions:
+      id-token: write
+      contents: read
+    name: Check if ECR images used as test dependencies exist
+    runs-on: ubuntu-latest
+    outputs:
+      latest-ethereum-client-image-exists: ${{ steps.check-latest-ethereum-client-image-exists.outputs.exists }}
+    steps:
+      - name: Checkout chainlink-testing-framework repository
+        uses: actions/checkout@v2
+        with:
+          repository: smartcontractkit/chainlink-testing-framework
+          path: chainlink-testing-framework
+      - name: Set up Go
+        uses: actions/setup-go@v3
+      - name: Run get-latest-ethereum-client-version ethereum/client-go:latest_stable command
+        id: get-latest-ethereum-client-version
+        run: |
+          cd chainlink-testing-framework/
+          ls utils/cmd/
+          version=$(go run utils/cmd/main.go get-latest-ethereum-client-version ethereum/client-go:latest_stable)
+          echo "version=${version}" >> $GITHUB_OUTPUT
+      - name: Check if latest Ethereum client image ${{ steps.get-latest-ethereum-client-version.outputs.version }} exists in ECR
+        id: check-latest-ethereum-client-image-exists
+        uses: smartcontractkit/chainlink-github-actions/docker/image-exists@b49a9d04744b0237908831730f8553f26d73a94b # v2.3.17
+        with:
+          repository: ethereum/client-go
+          tag: ${{ steps.get-latest-ethereum-client-version.outputs.version }}
+          AWS_REGION: ${{ secrets.QA_AWS_REGION }}
+          AWS_ROLE_TO_ASSUME: ${{ secrets.QA_AWS_ROLE_TO_ASSUME }}
+      - name: Latest Ethereum client image does not exist in ECR
+        if: steps.check-latest-ethereum-client-image-exists.outputs.exists != 'true'
+        run: |
+          echo "ethereum/client-go:${{ steps.get-latest-ethereum-client-version.outputs.version }} does not exist in our ECR. The image download was scheduled, and you can rerun this workflow in few minutes."
+          exit 1          
```

Comment:
this is not a good idea, why can't we wait for mirror update job to finish?

Reply:
Fixed